doc/Type/Cool.pod6

```diff
@@ -1494,6 +1501,119 @@ string|/routine/sprintf#Directives>, to format and print the arguments
 =for code
 "%.8f".printf(now - now ); # OUTPUT: «-0.00004118»
 
+=head2 method Complex
+
+Defined as:
+
+    multi method Complex()
+
+Coerces the invocant to a L«C<Numeric>|/type/Numeric» and calls its
+L«C<.Complex>|/routine/Complex» method. L<Fails|/routine/fail> if the
+coercion to a C<Numeric> cannot be done.
+
+    say 1+0i.Complex;         # OUTPUT: «1+0i␤»
+    say 2e1.Complex;          # OUTPUT: «20+0i␤»
+    say 1.3.Complex;          # OUTPUT: «1.3+0i␤»
+    say (-4/3).Complex;       # OUTPUT: «-1.3333333333333333+0i␤»
+    say "foo".Complex.^name;  # OUTPUT: «Failure␤»
+    say 1+1i.Complex;         # OUTPUT: «1+1i␤»
+
+=head2 method FatRat
+
+Defined as:
+
+    multi method FatRat()
+
+Coerces the invocant to a L«C<Numeric>|/type/Numeric» and calls its
+L«C<.FatRat>|/routine/FatRat» method. L<Fails|/routine/fail> if the
+coercion to a C<Numeric> cannot be done.
+
+    say 1+0i.Rat;             # OUTPUT: «1␤»
+    say 2e1.Rat;              # OUTPUT: «20␤»
+    say 1.3.Rat;              # OUTPUT: «1.3␤»
+    say (-4/3).Rat;           # OUTPUT: «-1.333333␤»
+    say "foo".Rat.^name;      # OUTPUT: «Failure␤»
+
+=head2 method Int
+
+Defined as:
+
+    multi method Int()
+
+Coerces the invocant to a L«C<Numeric>|/type/Numeric» and calls its
+L«C<.Int>|/routine/Int» method. L<Fails|/routine/fail> if the
+coercion to a C<Numeric> cannot be done.
+
+    say 1+0i.Int;             # OUTPUT: «1␤»
+    say 2e1.Int;              # OUTPUT: «20␤»
+    say 1.3.Int;              # OUTPUT: «1␤»
+    say (-4/3).Int;           # OUTPUT: «-1␤»
+    say "foo".Int.^name;      # OUTPUT: «Failure␤»
+
+=head2 method Num
+
+Defined as:
+
+    multi method Num()
+
+Coerces the invocant to a L«C<Numeric>|/type/Numeric» and calls its
+L«C<.Num>|/routine/Num» method. L<Fails|/routine/fail> if the
+coercion to a C<Numeric> cannot be done.
+
+    say 1+0i.Num;             # OUTPUT: «1␤»
+    say 2e1.Num;              # OUTPUT: «20␤»
+    say 1.3.Num;              # OUTPUT: «1.3␤»
+    say (-4/3).Num;           # OUTPUT: «-1.3333333333333333␤»
+    say "foo".Num.^name;      # OUTPUT: «Failure␤»
```

Comment:
Maybe include an allomorph as an example?

Reply:
You mean for all these coercers or specificaly for `.Num`  ?

Reply:
Well, this comment was about Num, but it would be good to have it anywhere it makes sense.